src/sentry/api/endpoints/organization_tagkey_values.py

```diff
@@ -22,10 +22,12 @@ def get(self, request, organization, key):
         except OrganizationEventsError as exc:
             return Response({'detail': exc.message}, status=400)
 
-        query = 'tags_key:%s' % (key,)
+        column_name = get_snuba_column_name(key)
+        # TODO: Make this work... we want column_name LIKE %query%
+        # query = request.GET.get('query', '')
 
         try:
-            snuba_args = get_snuba_query_args(query, params=filter_params)
+            snuba_args = get_snuba_query_args('', params=filter_params)
```

Comment:
i think you still want there to be a query passed here so that you can filter to make sure that tag key / column is not null